docs/sql-guide/functions/function-setexcludes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SETEXLCUDES()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# SETEXCLUDES() function
+
+`SETEXCLUDES()` tests membership of a set of values within a set. It returns true if all of the members of `testset` do not exist in `targetset`
+
+## Syntax
+
+```
+setexcludes(targetset, testmember)
+```
+
+## Arguments
+
+| Argument | Description | Data type |
+|---|---|---|
+| `targetset` | The set in which the members of testset are being tested for membership. | `stringset` or `idset` |
```

Comment:
Based on the examples, `targetset` is actually the column name, correct?
Instead of `targetset` I'd call this `set-column` because it's clearer just by reading.

Reply:
Based on the examples, I'd also wonder if the `testmember` value can be a comma-separated list or if each value always needs to be identified with the function?

Reply:
`testmember` cannot be a comma separated list per @paddyjok, you have to create a union by adding `AND` creating an setexludes() per value to exclude, I added an example at the end for this purpose